src/services/orderService.ts

```diff
@@ -0,0 +1,24 @@
+import useSWR, { mutate } from 'swr';
+
+const orderProducts = async (cartItems: any) => {
```

Comment:
mały hint - nie nazywałbym funkcji od order tylko make Order / create Order. bo tak to wygląda jak funkcja do ustalania kolejności czy coś